terraform/variables.tf

```diff
@@ -114,20 +114,30 @@ variable "enable_confidential_nodes" {
 }
 
 variable "acm_version" {
-  description = "Anthos Config Management version"
+  description = "Config Management version"
```

Comment:
Is it Config Sync instead ?

Reply:
The provider still says ACM: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/gke_hub_feature_membership#version